pkg/client/client_repository.go

```diff
@@ -92,3 +92,36 @@ func (c *Client) RepositoryInfo(owner string, name string) (*RepositoryInfo, err
 
 	return info, nil
 }
+
+type repositoriesNamesQuery struct {
+	RateLimit       rateLimit
+	RepositoryOwner struct {
+		Repositories struct {
+			Nodes []struct {
+				Name string
+			}
+		} `graphql:"repositories(last: 100, isFork: false, isLocked: false, affiliations: OWNER)"`
+	} `graphql:"repositoryOwner(login: $login)"`
+}
+
+func (c *Client) RepositoriesNames(login string) ([]string, error) {
```

Comment:
We fetch only the 100 first repositories ? Not all ?

Reply:
Calling this API is limited to 100 items. We should use pagination. In the next PR, after we get some dashboards ?

Reply:
Seems reasonable for me (since the limitation is well mentioned in the command line).

Reply:
Ok for me